cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/utils/NGSICharsets.java

```diff
@@ -30,19 +30,153 @@ private NGSICharsets() {
     } // NGSICharsets
     
     /**
-     * Encodes a string for CartoDB.
+     * Encodes a string for PostgreSQL. This includes CartoDB. Only lowercase alphanumercics and _ are allowed.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1133/commits/3ba31a420f9ddda138c790c11f4d8fdc426ba8f4